src/redux/categories/categories.js

```diff
@@ -0,0 +1,19 @@
+// Actions
+const CHECKSTATUS = 'bookstore/categories/checkStatus';
+
+// Reducer
+export default function categoriesReducer(state = [], action = {}) {
+  switch (action.type) {
+    case CHECKSTATUS:
+      return 'Under construction';
+    default: return state;
+  }
+}
+
+// Action Creators
+export function checkStatus(id) {
+  return {
+    type: CHECKSTATUS,
+    payload: { id },
+  };
```

Comment:
- [ ] Kindly make sure to remove the `id` because it is not being used anywhere in the reducer.  The action can be created without `payload` but it must have `type`. I highly recommend deleting `payload` because it is being used in the reducer.